docs/source/configuration.rst

```diff
@@ -152,6 +152,48 @@ as lists, dictionaries, and so on with normal Python syntax.
 Environment variables take precedence over configuration values found in YAML
 files.
 
+Conversion Utility
+^^^^^^^^^^^^^^^^^^
+
+Many examples in the documentation use YAML config in their examples. You can enter
```

Comment:
What do you think about using the text you added over in the configuration reference here? It seems a little more clear to me that there are several ways to set config values and this utility helps you convert between them. 

```suggestion
It is possible to configure Dask inline with dot notation, with YAML or via environment variables. You can enter
```